packages/Toast/src/Toast.styles.js

```diff
@@ -86,7 +86,7 @@ export const Toast = styled.div(
     font-weight: normal;
     margin-bottom: ${stylers.spacer(2)};
     min-height: ${stylers.spacer(3)};
-    padding: ${tokens.spaceLg};
+    padding: ${hasCloseButton ? "7px" : "11px"};
```

Comment:
This makes the alignment inconsistent between `hasCloseButton==true` and `hasCloseButton==false` – namely the left padding. I think for both the left padding (and probably the right too) should consistently be `12px` (`spaceLg`).

Reply:
Design needs the toast to be 48px in height, and close button to be bigger (32px). In order to maintain these request. I need to remove some padding when the close button is added. The other way to do it is to add negative margin on close button. Do you want me to do that instead?

Reply:
Changing the top and bottom padding is fine, but since the CSS rule is for `padding` it's changing the left and right padding as well, which we don't want to do.

Reply:
sorry I purposely change the padding to be the same on four sides. I will reverse that change.